be1-go/message/messagedata/transaction_post.go

```diff
@@ -0,0 +1,53 @@
+package messagedata
+
+//TBC
+type TransactionPost struct {
+	Object      string            `json:"object"`
+	Action      string            `json:"action"`
+	Transaction TransactionStruct `json:"transaction"`
+}
+
+//TBC
+type TransactionStruct struct {
+	Version  int           `json:"version"`
+	TxIn     []TxInStruct  `json:"txin"`  // min 1
+	TxOut    []TxOutStruct `json:"txout"` //min 1
```

Comment:
Why not just `Input` and `Output` instead of `TxIn` and `TxOut`?

Reply:
I've used TxIn and TxOut because we named them like that in the documentation. 
Tx stands for transaction in the bitcoin specifications.

Reply:
I know, but it seems that those express the Input/outputs of the transaction. Since your are defining an attribute of a Transaction, then if you mention Input/Output within the Transaction definition, you're already saying this is the Input/Output of the transaction. No need to mention it in the attributes' name.